extras/tools/delete_unowned_bricks.sh

```diff
@@ -0,0 +1,266 @@
+#!/bin/bash
+
+set -o pipefail
+
+dryRun="no"
+lvlist=""
+prompt="no"
+vg_id=""
+ip_prefix=""
+fstab_path=""
+
+
+usage() {
+    echo "Usage: $0 [...]"
+    echo ''
+    echo 'The options that this script accepts are:'
+    echo ''
+    echo '--dryrun          shows the commands that would be executed'
+    echo '--lvlist          has the list of volumes to delete'
+    echo '--prompt          prompts for user input before executing every command'
+    echo '--vgid            vgid containing lvs'
+    echo '--ip-prefix       IP address prefix'
+    echo '--fstab-path      provide the path for the fstab file'
+    echo '-h, --help        show this help text'
+    echo ''
+}
+
+
+function parse_args () {
+    args=$(getopt \
+              --options h \
+              --long dryrun,lvlist:,vgid:,prompt,ip-prefix:,fstab-path:,help \
+              -n 'delete_unowned_bricks.sh' \
+              --  "$@")
+    eval set -- "$args"
+    while true; do
+        case "$1" in
+            --dryrun) dryRun="yes"; shift ;;
+            --prompt) prompt="yes"; shift ;;
+            --ip-prefix) ip_prefix=$2; shift 2 ;;
+            --fstab-path) fstab_path=$2; shift 2 ;;
+            --lvlist)
+                case "$2" in
+                    "") shift 2 ;;
+                     *) lvlist="$2" ; shift 2 ;;
+                esac ;;
+            --vgid)
+                case "$2" in
+                    "") shift 2 ;;
+                     *) vg_id="$2" ; shift 2 ;;
+                esac ;;
+            -h|--help) usage ; exit 0 ;;
+            --) shift ; break ;;
+            *)
+                echo "failed to parse cli (error at $1)"
+                exit 1
+            ;;
+        esac
+    done
+}
+
+precmd() {
+    echo "RUN CMD: $*" >&2
+    if [[ "$dryRun" = "yes" ]]; then
+        return 0
+    fi
+    if [[ "$prompt" = "yes" ]]; then
+        read -r -p 'Continue? [y/N]: ' pval
+        if ! echo "$pval" | grep -q -i '^y'; then
+            return 0
+        fi
+    fi
+    "$@"
+}
+
+function checkLocalNodeHasGlusterdRunning() {
+    # Check if local node has gluster binary and if glusterd is running
```

Comment:
Just a cosmetic thing: wouldn't it be better to have the comment block *before* the function?

Reply:
removed the comment. Not useful anymore.